server/Controllers/movie.js

```diff
@@ -65,6 +65,8 @@ const receiveReviews = (req, res) => {
           _id: new mongoose.Types.ObjectId(),
           movieId,
           name: req.body.name,
+          // Put Movie name here in the future
```

Comment:
add `todo: ` before (to allow us to find it easily)
```suggestion
          // todo: Put Movie name here in the future
```